src/main/java/dev/ebullient/convert/tools/dnd5e/Tools5eJsonSourceCopier.java

```diff
@@ -222,8 +222,8 @@ protected JsonNode mergeNodes(Tools5eIndexType type, String originKey, JsonNode
 
                 JsonNode templateMods = normalizeMods(MetaFields._mod.getFrom(apply));
                 if (templateMods != null) {
-                    if (_mod == null) {
-                        _mod = templateMods;
+                    if (!MetaFields._mod.existsIn(_copy)) {
+                        MetaFields._mod.setIn(_copy, templateMods);
                     } else {
```

Comment:
This was very sticky/tricky stuff that was not easy to find. AND.. the way I did this mattered. I'll have to think about why, but the assignment of the variable in this place in this way mattered (and is different than using MetaFields to set it in the node. I'll have to study it to remember

Reply:
looks like the _template version works differently anyway, so it might be a moot point for now